.github/workflows/reset-workflow-timeout.yaml

```diff
@@ -0,0 +1,17 @@
+name: Keep scheduled jobs running
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *"
+
+# Prevent scheduled jobs being disabled after 60 days
+# see https://github.community/t/no-notification-workflow-disabled-after-60-days/182169
+# existing GitHub notifications don't seem to work at all
+# this should trigger a dummy commit to the default branch every 50 days
```

Comment:
looks like v2 uses the API to keep it alive in someway by default and doesn't need to commit

Reply:
Updated to v2 as it was in the original change.